src/Uno.UWP/Security/Authentication/Web/WebAuthenticationBrokerProvider.cs

```diff
@@ -42,7 +48,11 @@ public Uri GetCurrentApplicationCallbackUri()
 				this.Log().Warn(message);
 			}
 
-			return new Uri(schemes[0] + WinRTFeatureConfiguration.WebAuthenticationBroker.DefaultCallbackPath);
+			var uri = new Uri(schemes[0] + WinRTFeatureConfiguration.WebAuthenticationBroker.DefaultCallbackPath);
+#if __ANDROID__
+			uri = uri.TrimEndUriSlash();
+#endif
+			return uri;
```

Comment:
Why not optimize it this way then?
```suggestion
            var scheme = schemes[0];
            var hostAndPath = WinRTFeatureConfiguration.WebAuthenticationBroker.DefaultCallbackPath;
            var uriStringBuilder =
                new StringBuilder(scheme.Length + hostNadPath.Length)
                    .Append(scheme);
                    .Append(hostAndPath);
#if __ANDROID__
            if (uriStringBuilder[^1] == '/')
			    uriStringBuilder.Lenght -= 1;
#endif
			return new Uri(uriStringBuilder.ToString());
```

Reply:
@carldebilly , do you see a needs todo implement in this way for now?

Reply:
That's not required for now.